cmd/azmount/filemanager/azure.go

```diff
@@ -91,30 +93,44 @@ func AzureSetup(urlString string, urlPrivate bool, identity common.Identity) err
 	}
 
 	if urlPrivate {
-		// we use token credentials to access private azure blob storage the blob's
-		// url Host denotes the scope/audience for which we need to get a token
-		logrus.Trace("Using token credentials to access private azure blob storage...")
-
-		var token common.TokenResponse
-		count := 0
-		logrus.Debugf("Getting token for https://%s", u.Host)
-		for {
-			token, err = common.GetToken("https://"+u.Host, identity)
-
+		ctx := context.TODO()
```

Comment:
Would it be worth adding a timeout to `ctx`?

Reply:
done